styleguide/components/11-base/fonts/_fonts.scss

```diff
@@ -2,5 +2,5 @@
 /// This file defines fonts.
 ////
 
-@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,600,700'); // sass-lint:disable-line no-url-protocols
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700'); // sass-lint:disable-line no-url-protocols
```

Comment:
Nooooooooooooooooo..... 😨 Why 😭

Reply:
Because it's in the design :/ I can't keep asking the designer for every little change :P

Reply:
We can, we are a team. There's no harm in adding your expertise to the decision making progress, that's how we learn.

It's been decided to keep font loading as low as possible.
Maybe design is not aware of the consequences off adding a weight, maybe design did not know this weight is not already included...

Maybe design did know and consider these things and found it important enough to add this weight, in which case we'll follow.

Either way, we need to know for sure.

Reply:
Okay, agreed, it goes back to impediment.